cmd/katib-controller/v1alpha2/main.go

```diff
@@ -20,49 +20,70 @@ limitations under the License.
 package main
 
 import (
-	"log"
+	"flag"
+	"os"
 
-	"github.com/kubeflow/katib/pkg/api/operators/apis"
-	controller "github.com/kubeflow/katib/pkg/controller/v1alpha2"
+	"github.com/spf13/viper"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	"github.com/kubeflow/katib/pkg/api/operators/apis"
+	controller "github.com/kubeflow/katib/pkg/controller/v1alpha2"
+	"github.com/kubeflow/katib/pkg/controller/v1alpha2/consts"
 )
 
 func main() {
 	logf.SetLogger(logf.ZapLogger(false))
+	log := logf.Log.WithName("entrypoint")
+
+	var fakeExperimentSuggestion bool
+
+	flag.BoolVar(&fakeExperimentSuggestion, "fake-experiment-suggestion",
+		false, "Use the fake suggestion in experiment controller")
```

Comment:
how about use args like `experiment-suggestion-name`, and now its value can be "fake|default", `default` by default. in future, we can may support more.

Reply:
Good point. I will update it.

Reply:
Addressed